src/Tribe/Tickets_Handler.php

```diff
@@ -789,4 +792,19 @@ public function get_attendee_report_link( $post ) {
 		return $url;
 	}
 
+	/**
+	 * Sets the `$typenow` var to the post_type
+	 *
+	 * Causes parent post type to be highlighted in admin menus.
+	 *
+	 * @see `admin_head` action
+	 */
+	public function set_typenow() {
+		global $typenow;
+
+		if ( ! empty( $_REQUEST['post_type'] ) ) {
```

Comment:
Why not use `tribe_get_request_var()`? Avoid usage of `_COOKIE`, will only check `_GET` and `_POST`

Reply:
WordPress itself sets `$typenow` based on `$_COOKIE` as well. As mentioned in the description I actually copied that line right from WP core (  /wp-admin/admin.php:124 ) Either way I will switch to tribe_get_request_var(), even at the risk of introducing a bug. I doubt this gets set via cookies very often.